exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,36 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10){
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isShortName(name) {
+  if (name === "Daniel"){
+    return true;
+  } else {
+    return false;
+  }
+
+}
+
+function startsWithD(nameStartWithD) {
```

Comment:
a parameter of name would be more appropriate here, as the function's name describes the functions action.